vector-config/src/tchap/res/values/config-features.xml

```diff
@@ -2,4 +2,5 @@
 <resources>
     <bool name="tchap_is_cross_signing_enabled">true</bool>
     <bool name="tchap_is_key_backup_enabled">true</bool>
+    <bool name="tchap_is_thread_enabled">false</bool>
```

Comment:
IMO you don't need to add `tchap_is_thread_enabled` boolean. You should use `settings_labs_thread_messages_default` instead.